src/main/webapp/app/assessment/assessment-detail/assessment-correction-round-badge/assessment-correction-round-badge.component.html

```diff
@@ -0,0 +1,9 @@
+<div>
+    <!-- correction round label-->
+    <h5 class="correction-label" *ngIf="highlightDifferences && feedback.copiedFeedbackId">
+        <span class="badge text-white mr-1" [ngStyle]="{ backgroundColor: '#3e8acc' }">First correction round</span>
+    </h5>
+    <h5 class="correction-label" *ngIf="highlightDifferences && !feedback.copiedFeedbackId">
+        <span class="badge text-white mr-1" [ngStyle]="{ backgroundColor: '#ffa561' }">Second correction round</span>
```

Comment:
I would use `jhiTranslate` instead of plain text

Reply:
good catch, thanks!